RELEASE-NOTES.txt

```diff
@@ -8,6 +8,7 @@
 * [*] Fixed a rare crash on the featured image confirmation dialog [https://github.com/wordpress-mobile/WordPress-Android/pull/20836]
 * [*] Fixed an ANR issue on the Post List screen [https://github.com/wordpress-mobile/WordPress-Android/pull/20833]
 * [*] [internal] Block Editor: Upgrade target sdk version to Android API 34 [https://github.com/wordpress-mobile/WordPress-Android/pull/20841]
+* [*] In-app updates feature [https://github.com/wordpress-mobile/WordPress-Android/pull/20822]
```

Comment:
We should the [internal] tag at the starting of the line

* [*] [internal] In-app updates feature [https://github.com/wordpress-mobile/WordPress-Android/pull/20822]

Reply:
I have done it in a commit. Hope its ok with you. Approving the PR. Leaving it for you to merge